src/com/zestedesavoir/zestwriter/view/Editor.fxml

```diff
@@ -141,11 +142,11 @@
             </ToolBar>
          </top>
          <center>
-            <TextArea fx:id="SourceText" wrapText="true" BorderPane.alignment="CENTER">
-               <font>
+            <CodeArea fx:id="SourceText" wrapText="true" BorderPane.alignment="CENTER">
+<!--               <font>
                   <Font name="Fira Mono" size="13.0" />
-               </font>
-            </TextArea>
+               </font>-->
```

Comment:
pas supprimé car WIP, mais commenté car théoriquement la CodeArea 
possède déjà une font spéciale.

Le 30/03/2016 22:25, WinXaito a écrit :
>
> In src/com/zestedesavoir/zestwriter/view/Editor.fxml 
> <https://github.com/firm1/zest-writer/pull/37#discussion_r57957412>:
>
> >                    <Font name="Fira Mono" size="13.0" />
> > -               </font>
> > -            </TextArea>
> > +               </font>-->
>
> Je ne comprend pas bien pourquoi avoir commenté ça ? (Et s'il doit 
> l'être, pourquoi n'est t'il pas supprimé ?)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/firm1/zest-writer/pull/37/files/b0773f6a395e826a2c3547bb7b41deb379356cf6..be027d8366fad0200843cf3dd670461e5a1ad3d9#r57957412>
>